### PR TITLE
Fix/navigation bar styling

### LINKED ISSUE #13: update Navigation bar visuals

### PR DESCRIPTION
**Pull request by AmineTir**
Take a look at the breadcrumb style, I tried to match it enough to windows explorer. The control buttons are not yet styled.

### CODE REVIEW

File: src/app/app.css [hunk 1]
```diff
@@ -23,3 +23,8 @@ html, body, #root, .app  {
   display: flex;
   flex-direction: column;
 }
+
+body  {
+  font-family: "Tahoma", sans-serif;
```
Comment (atahrijouti) [inline src/app/app.css]: Is it the same font that's used by windows 10?

File: src/navigation-bar/navigation-bar.css [hunk 2]
```diff
@@ -4,27 +4,39 @@
...
   background: white;
   display: flex;
+  border: 1px solid #e2e2e2;
 }
 
 .navigation-bar .buttons {
   flex: none;
 }
 
+.navigation-bar .buttons>button {
```
Comment (atahrijouti) [inline src/navigation-bar/navigation-bar.css]: Over specifying is bad. Maybe give the button a class which you could target with `.navigation-bar .go-up`?

File: src/app/app.css (hunk 1 above)
Comment (AmineTir) [reply src/app/app.css]: I don't know the exact font used in W10, so I chose "Tahoma" for now.

File: src/navigation-bar/index.ts [hunk 3]
```diff
@@ -87,12 +91,12 @@ export function NavigationBar() {
...
   }
 
-  const goUp = h("button", "⬆️")
+  const goUp = h("button", "🡱")
   const breadcrumb = h("ul", { className: "breadcrumb" })
-  const newFolderBtn = h("button", "New Folder")
-  const newFileBtn = h("button", "New File")
-  const renameNodeBtn = h("button", { disabled: true }, "Rename")
-  const deleteNodeBtn = h("button", { disabled: true }, "Delete")
+  const newFolderBtn = h("button", h("img", { src: folderImage }), h("span", "New"), h("br"), h("span", "Folder"))
```
Comment (atahrijouti) [inline src/navigation-bar/index.ts]: This suggests to me that we might need to create s new component that creates control bar buttons. But we can leave it for another pr, so as to not kill your flow

File: src/app/app.css (hunk 1 above)
Comment (atahrijouti) [reply src/app/app.css]: Then research time :P

File: src/navigation-bar/index.ts (hunk 3 above)
Comment (atahrijouti) [reply src/navigation-bar/index.ts]: For now give the button a class of `control-button` and style that instead of ‘... > button`
